CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # Changelog
 
+## [1.27.0] - 2024-01-29
+
+This long-awaited Rustup release has gathered all the new features and fixes since April 2023.
+These changes have greatly improved Rustup's maintainability, user experience, compatibility and documentation quality.
+
+The headlines of this release are:
+- Basic support for `fish` shell has been added.
+- Rustup now uses `clap` v4 for argument parsing.
```

Comment:
I would suggest we do not mention the clap upgrade and the GitHub merge queue changes here, since they aren't really relevant to our users.

Reply:
Will do.

Reply:
Removed from the headlines. I do want to mention the clap v4 thing here since the CLI output is slightly changed. Although there's no clear evidence whether it is popular to parse Rustup's output, I think it's better to at least leave a hint.